static/index.html

```diff
@@ -33,14 +34,32 @@ <h3 id="title"></h3>
         let stateURL = new URL(window.location.pathname + "state", window.location.origin)
         let stopURL = new URL(window.location.pathname + "stop", window.location.origin)
         let playButton = document.getElementById("play");
+        let loopButton = document.getElementById("loop");
+        let loopBool = false;
         playButton.addEventListener("click", Play);
-   
+        loopButton.addEventListener("click", toggleBool);
+
+        async function toggleBool(){
+            loopBool = !loopBool;
+            if(loopBool){
+                loopButton.textContent = "Video will loop!";
+                loopButton.style.backgroundColor="darkblue";
+            }
+            else{
+                loopButton.textContent = "Loop Video?";
+                loopButton.style.backgroundColor="blue";
+            }
+        }
+        
         async function Play(e) {
             e.preventDefault();
             const url = document.getElementById('url').value;
             let playURL = new URL(window.location.pathname + "play", window.location.origin)
-
+            
             playURL.searchParams.append("url", encodeURIComponent(url))
+            if(loopBool){
+                playURL.searchParams.append("loop","True")
```

Comment:
as long as something is in the param the api should loop, the below works

```suggestion
                playURL.searchParams.append("loop", 1)
```

Reply:
done